srcpkgs/synfigstudio/template

```diff
@@ -11,7 +11,7 @@ maintainer="newbluemoon <blaumolch@mailbox.org>"
 license="GPL-2.0-or-later"
 homepage="http://www.synfig.org/"
 distfiles="https://github.com/synfig/synfig/releases/download/v${version}/synfigstudio-${version}.tar.gz"
-checksum=af9a9b0064064907ae24a693b607dcba633a8a2cd45fa1357f61c035efd6020e
+checksum=4e86d4d8def6675fc92855a411795cc4062a1669cda82535d18b48a68b4d5497
 python_version=3
 CXXFLAGS="-Wno-deprecated-copy -DHAVE_LOCALE_H=1"
 
```

Comment:
`synfig` should be put in `hostmakedepends` regardless of cross.

Reply:
Also: `export DATADIRNAME=share`, otherwise, you'll see `-DLOCALEDIR=/usr//locale`

Reply:
However, when checking the built packages (native and cross) the localization files are always installed correctly in `/usr/share/locale`. Shall I explicitly add `DATADIRNAME` anyway?

Reply:
It's installed correctly, but load incorrectly. So, yes, please add `DATADIRNAME`.